banner.php

```diff
@@ -13,7 +13,21 @@ function should_show_banner() {
 	}
 
 	$default = -1;
-	return get_option( WIKIPEDIA_PREVIEW_BANNER_OPTION, $default ) === $default;
+	$value   = get_option( WIKIPEDIA_PREVIEW_BANNER_OPTION, $default );
+
+	if ( $value === $default ) {
+		// never shown
```

Comment:
i think you mean always shown the banner here.

Reply:
What I meant was that the banner was never shown before therefore it should be shown now. But in fact, the condition means it was never dismissed. I'll reword the comment.